source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -561,21 +561,13 @@ async function readInlangConfig(args: {
 			$fs: args.fs.promises,
 		}
 		const file = await args.fs.promises.readFile("./inlang.config.js", "utf-8")
-		const withMimeType = "data:application/javascript;base64," + btoa(file.toString())
+		const withMimeType =
+			"data:application/javascript;base64," + Buffer.from(file).toString("base64")
```

Comment:
`btoa` is available in every JS runtime. `Buffer` is a node dependency. The preview link works though https://inlang-website-pr-473.onrender.com/editor/github.com/inlang/example. 

Thus, can stay as is. Implementing your ESLint no-node dependencies would likely be a win.

Reply:
Ah right, this issue again ^^
I was confused by the deprecated flag and did want to fix it.

The problem here is that if any of our dependencies imports node types globally, we will have them in our project.
https://stackoverflow.com/a/67431122/9832456

We can't really get rid of the types unless we remove that dependency or that dependency releases an update which fixes the issue..

I have reverted those changes